libs/ui-lib/lib/common/api/axiosClient.ts

```diff
@@ -36,14 +37,11 @@ let client = applyCaseMiddleware(
   },
 );
 let clientWithoutCaseConverter = withAssistedInstallerBasePath(axios.create());
-
-export const setAuthInterceptor = (authInterceptor: (client: AxiosInstance) => AxiosInstance) => {
-  isInOcm = true;
+//I need to figure out how to know if we are in OCM or not
+if (isInOcm) {
```

Comment:
Let OCM tell you that... :) For that you'll need to create some sort of abstraction for the ui-lib configuration.
Let's discuss this in a 1-1.

Reply:
@jkilzi can we discuss this ?